src/main/java/chess/domain/board/Board.java

```diff
@@ -0,0 +1,163 @@
+package chess.domain.board;
+
+import static chess.domain.Team.NEUTRALITY;
+
+import chess.domain.Position;
+import chess.domain.Team;
+import chess.domain.math.Direction;
+import chess.domain.math.UnitVector;
+import chess.domain.pieces.Bishop;
+import chess.domain.pieces.EmptyPiece;
+import chess.domain.pieces.King;
+import chess.domain.pieces.Knight;
+import chess.domain.pieces.Pawn;
+import chess.domain.pieces.Piece;
+import chess.domain.pieces.Queen;
+import chess.domain.pieces.Rook;
+import java.util.List;
+
+public class Board {
+
+    private final List<Rank> board;
+
+    private Board(List<Rank> ranks) {
+        this.board = ranks;
+    }
+
+    public static Board create() {
+        return new Board(BoardMaker.create());
+    }
+
+    public void movePiece(Position current, Position target) {
+        Piece currentPointPiece = findPiece(current);
+        Direction moveableDirection = Direction.findDirection(current, target);
+
+        if (currentPointPiece.hasDirection(moveableDirection)) {
+            runLogic(current, target, moveableDirection);
+            return;
+        }
+        throw new RuntimeException("[ERROR] 패턴에서 걸러진 예외");
```

Comment:
```suggestion
        if (currentPointPiece.hasNoDirection(moveableDirection)) {
            throw new RuntimeException("[ERROR] 패턴에서 걸러진 예외");
        }
        
        runLogic(current, target, moveableDirection);
```

요게 더 자연스러운 흐름으로 보이네요 😊